### PR TITLE
Fix: Allow blocked comments with markers and new-line (fixes #2777)

### LINKED ISSUE #2777: `spaced-comment` warns on "don't minify" comment blocks
Typically, the first comment block in a file starts with `/*!` to indicate to a minifier like uglify that it should not strip the comment block (a "prelude").

Could we add an option to the `spaced-comment` rule that allows this pattern?

### PR DESCRIPTION
**Pull request by gyandeeps**

### CODE REVIEW

**Comment by gcochard** [comment]
I think a better fix for this is to split `node.value` on newline if `node.type === "Block"` and then run the marker test on each line (or only the first line) of the comment. Thoughts?

**Comment by gyandeeps** [comment]
@gcochard I am open for suggestions. Can you share an example for which we need the strategy you discussed above?

**Comment by nzakas** [comment]
@gcochard I'm not sure splitting gives any particular advantage here. It would introduce a code fork based on comment type where one isn't really needed now.

**Comment by gcochard** [comment]
I had an assumption that it might pass as valid on something like this:

Options:
```json
options: ["always", { "markers": ["!"] }]
```
Code:
```js
/*nospaceshere
! hooray a space
*/
```

Testing in the node REPL has proved that assumption wrong.

Looking forward however, I can foresee a feature request with this rule to expand to each line of multi-line comments. Let's wait for that to happen before adding any complexity.

TL;DR version:
:+1: LGTM, :shipit:.

**Comment by nzakas** [comment]
Cool, we will go with this for now.
